src/coreclr/jit/emitxarch.cpp

```diff
@@ -20056,6 +20056,7 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
         case INS_serialize:
         {
             result.insThroughput = PERFSCORE_THROUGHPUT_50C;
+            result.insLatency = PERFSCORE_LATENCY_105C;
```

Comment:
Looks like there's an early formatting failure, I think it needs to be this (but the format patch in the artifacts would say for certain):
```suggestion
            result.insThroughput = PERFSCORE_THROUGHPUT_50C;
            result.insLatency    = PERFSCORE_LATENCY_105C;
```